pwa/src/templates/templateParts/download/DownloadTemplate.tsx

```diff
@@ -1,63 +1,82 @@
 import * as React from "react";
 import * as styles from "./DownloadTemplate.module.css";
-
+import clsx from "clsx";
 import { DownloadCard, NotificationPopUp as _NotificationPopUp } from "@conduction/components";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faArrowLeft, faDownload } from "@fortawesome/free-solid-svg-icons";
 import { useTranslation } from "react-i18next";
-import clsx from "clsx";
 import { useGatsbyContext } from "../../../context/gatsby";
+import { TOOLTIP_ID } from "../../../layout/Layout";
 
 interface DownloadTemplateProps {
-  icon: JSX.Element;
-  label: string;
-  sizeKb: string;
+  items: DownloadProps[];
+  backUrl: string;
+  icon?: JSX.Element;
```

Comment:
In some implementations, we're passing a `FontAwesomeIcon` type (unsure the exact type), instead of a `JSX.Element`. I'd do that here as well (right now, we can add **anything** to this `icon`, which is prone to error).